test/quantization/test_backward_compatibility.py

```diff
@@ -69,6 +71,69 @@ def remove_prefix(text, prefix):
         self.assertEqual(qmodule_scripted(input_tensor), expected, atol=prec)
         self.assertEqual(qmodule_traced(input_tensor), expected, atol=prec)
 
+    def _test_op_graph(self, qmodule, subname=None, input_size=None, input_quantized=True,
+                       generate=False, prec=None, new_zipfile_serialization=False):
+        r"""
+        Input: a floating point module
+
+        If generate == True, traces and scripts the module and quantizes the results with
+        PTQ, and saves the results.
+
+        If generate == False, traces and scripts the module and quantizes the results with
+        PTQ, and compares to saved results.
+        """
+        # NB: we take __file__ from the module that defined the test
+        # class, so we place the expect directory where the test script
+        # lives, NOT where test/common_utils.py lives.
+        module_id = self.__class__.__module__
+        munged_id = remove_prefix(self.id(), module_id + ".")
+        test_file = os.path.realpath(sys.modules[module_id].__file__)
+        base_name = os.path.join(os.path.dirname(test_file),
+                                 "serialized",
+                                 munged_id)
+
+        subname_output = ""
+        if subname:
+            base_name += "_" + subname
+            subname_output = " ({})".format(subname)
+
+        input_file = base_name + ".input.pt"
+        state_dict_file = base_name + ".state_dict.pt"
+        scripted_module_file = base_name + ".scripted.pt"
+        traced_module_file = base_name + ".traced.pt"
+        expected_file = base_name + ".expected.pt"
```

Comment:
yeah, there is potential to reuse some part of this code, it didn't seem worth it at first glance as it would increase complexity and the benefit would be small as not all of this can be reused.  I can take another look if you prefer.

Reply:
I don't have a strong preference, I just feel in general we should not be copying code.